### PR TITLE
Friends

### LINKED ISSUE #3: Create "Friends" section
Add "Friends" nav button and show friends list in centarl pane

### PR DESCRIPTION
**Pull request by olegho**
resolve #3

### CODE REVIEW

File: html/friends.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div id="friends">
+    <h1><span id='friends'></span></h1>
```
Comment (drm89) [inline html/friends.html]: id attribute must be unique within one page

Comment (olegho) [inline html/friends.html:2]: id attribute must be unique within one page

File: js/base.js [hunk 2]
```diff
@@ -54,13 +55,13 @@ $(document).ready(
...
 function loadCentralPage(url, controllerName) {
-	if (url != null && url != "") {
-		$("article").load(url, function() {
-			if (controllerName != null && controllerName != "") {
-				VK_CLONE.controllers[controllerName]();
-			}
-		});
-	}
-}
+ 	if (url != null && url != "") {
```
Comment (olegho) [inline js/base.js:58]: fix formatting here and below

File: js/controllers.js [hunk 3]
```diff
@@ -6,5 +6,13 @@ VK_CLONE.controllers = {
 		} else {
 			message.text("Please, sing in.");
 		}
+	},
+	friends: function() {
+	    var message = $("#friends");
```
Comment (olegho) [inline js/controllers.js:11]: fix formatting here and below
use tabs as indent

File: js/user.js [hunk 4]
```diff
@@ -6,4 +6,8 @@ function User(vkUser) {
 	this.getFullName = function() {
 		return user.first_name + " " + user.last_name;
 	};
+	this.getFriends = function() {
```
Comment (olegho) [inline js/user.js:9]: friends shouldn't be a part of User object. Create separate js file for "friends" functionality where you'll wrap call to VK API

**Pull request closed without merging**